recipe/build_base.sh

```diff
@@ -180,8 +180,8 @@ if [[ "${CONDA_BUILD_CROSS_COMPILATION}" == "1" ]]; then
   echo "ac_cv_file__dev_ptc=no"         >> config.site
   echo "ac_cv_pthread=yes"              >> config.site
   echo "ac_cv_little_endian_double=yes" >> config.site
+  echo "ac_cv_aligned_required=no"      >> config.site
   if [[ "${target_platform}" == "osx-arm64" || "${target_platform}" == "linux-ppc64le" || "${target_platform}" == "linux-aarch64" ]]; then
-      echo "ac_cv_aligned_required=no" >> config.site
```

Comment:
This doesn't change anything right?

Reply:
Right, it is just trying to replicate a similar change in #719, which I think ended up adding `ac_cv_aligned_required=no` twice to `config.site`.

Reply:
If you prefer, I can remove this change.